### ISSUE TITLE
Notarize app releases for macOS Catalina (and beyond)

### ISSUE THREAD

**Issue by jtkiley** [tuxu/nbviewer-app#17]
Please consider submitting the app to be notarized by Apple for Gatekeeper.

While users can get around it by telling macOS to open it anyway, it would be nice if the app were notarized. From looking at [the developer ID page](https://developer.apple.com/developer-id/) and [the more detailed instructions](https://developer.apple.com/documentation/security/notarizing_your_app_before_distribution), it seems like a fairly simple step to add to a release checklist or even a script.

I often recommend the app to folks new to the Python data science ecosystem (especially for the quick view functionality), and this would eliminate one (small) hurdle for them.

**Comment by tuxu** [comment]
You're right, it would be fairly simple thing to do. But it requires me to enroll in the Apple Developer Program, which is $99 per year. I'm wondering how other free and open source projects deal with notarization?

**Comment by noah-nuebling** [comment]
> I'm wondering how other free and open source projects deal with notarization?

What @amadeuspzs said is right! 

One thing to add is that Mac Mouse Fix has a little banner at the top of the [official website](https://mousefix.org/) that says „If you have trouble opening the app click here”. It links to a small guide that shows people how to work around the lack of Notarization.

Before I had that in place I got lots of messages from people having trouble opening the app, but this seems to work pretty well.

**Comment by jtkiley** [comment]
Of all of the open source stuff I run, this is the single thing that isn't notarized. The reality is that it makes it frustrating for users, and it also throws up a scary warning for new users to the Jupyter ecosystem.

If you're not willing to do it, maybe see if the VS Code devs will take the quicklook plugin.
